job-server-api/src/main/scala/spark/jobserver/context/SparkContextFactory.scala

```diff
@@ -76,22 +77,23 @@ trait ScalaContextFactory extends SparkContextFactory {
 
   val logger = LoggerFactory.getLogger(getClass)
 
-  def loadAndValidateJob(appName: String,
-                         uploadTime: DateTime,
-                         classPath: String,
-                         jobCache: JobCache): J Or LoadingError = {
+  def loadAndValidateJob(
```

Comment:
I think it's easier to read the way it was even if it's not a multiple of 2 spaces (you can always move them just one space). It's a common practice to align parameters with the opening parenthesis.